src/libraries/System.Text.Json/tests/Common/PropertyNameTests.cs

```diff
@@ -494,5 +494,31 @@ public class ClassWithSpecialCharacters
             [JsonPropertyName("\uA000_2")] // Valid C# property name: \uA000_2
             public int YiIt_2 { get; set; }
         }
+
+        [Fact]
+        public async Task ClassWithIgnoredCaseSensitiveConflict_WorksAsExpected()
```

Comment:
Suggestion for a more descriptive test name:
```suggestion
        public async Task ClassWithIgnoredCaseSensitiveConflict_RespectsIgnoredMember()
```